src/uwtools/tests/drivers/test_forecast.py

```diff
@@ -207,7 +207,7 @@ def test_forecast_run_cmd():
 
 @pytest.mark.parametrize("section", ["static", "cycledep"])
 @pytest.mark.parametrize("link_files", [True, False])
-def test_stage_files(tmp_path, section, link_files):
+def test__stage_files(tmp_path, section, link_files):
```

Comment:
This is still a public method in the forecast driver as far as I can tell.

Reply:
Will fix, thanks!